docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4385,31 +4379,35 @@ P4Runtime supports batching of `Write` operations. The list of updates in a
 updates on an arbitrary set of P4 entities. It is not restricted to a particular
 entity or table (in the case of `TableEntry` entities).
 
-The P4Runtime server may arbitrarily reorder message within a batch to maximize
-performance, and clients should not depend on a specific processing order (&eg;
-FIFO or inferring implicit dependencies within a batch). In particular, P4
-entities (&eg; table entries) may be inserted in the data plane in an order
-different than what is received in the `WriteRequest`.
+The P4Runtime server may choose to reorder updates in a batch when processing
+them, and/or process updates in parallel.  Updates across multiple concurrent
+`WriteRequest`s can also be processed interleaved and/or in parallel.
+However, **the processing of updates must be strictly serializable**.  That
+is, given a set $S$ of `WriteRequest`s including their responses to those
+requests, there must exist an order $L$ for all updates in $S$, such that:
```

Comment:
sorry, I missed that you were including the responses in `S`. However, I don't think that "including their responses to those requests" makes sense. What is "theirs" referring to? Seems that it should be either "including their responses" or "including the responses to those requests".

Reply:
Agreed, fixed.